docs/contributing-to-analyses/analysis-modules/notebook-structure.md

```diff
@@ -0,0 +1,150 @@
+# Structuring analysis notebooks
+
+Notebook formats such as [R Markdown](https://rmarkdown.rstudio.com/index.html) or [Jupyter](https://jupyter-notebook.readthedocs.io/en/latest/) are a great way to document an analysis and share it with others.
+We have found that following some common patterns can enhance sharing and reproducibility, so we suggest the following overall structure when you create a notebook:
+
+- **Headers**: Start with a header that includes a title, author, and date.
+- **Introduction**: Briefly describe the analysis and its purpose.
+- **Setup**: Load required packages and define paths for input and output files.
+- **Functions**: Define any custom functions used in the analysis.
+- **Analysis**: This is where the bulk of the analysis should appear.
+  - Intersperse code, to do the work, and text, to explain the steps and interpret results.
+  - Use headings and subheadings to break up the analysis into logical sections.
```

Comment:
might need more spaces but I think this will do it
```suggestion
    - Intersperse code, to do the work, and text, to explain the steps and interpret results.
    - Use headings and subheadings to break up the analysis into logical sections.
```